pkg/daemon/ceph/client/mirror.go

```diff
@@ -85,15 +86,27 @@ func ImportRBDMirrorBootstrapPeer(context *clusterd.Context, clusterInfo *Cluste
 		return err
 	}() //nolint // we don't want to return here
 
+	content, err := ioutil.ReadFile(tokenFilePath.Name())
+	if err != nil {
+		return fmt.Errorf("failed to read local file: %w", err)
+	}
+
+	command := fmt.Sprintf("echo '%s' > %s", string(content), tokenFilePath.Name())
```

Comment:
This contains the use key, what you need to ensure is that you will not log the command and also you need to ensure is that in worst case the command is not part of error (if something goes wrong) or else we are going to get security issues.  or else we need to use shared volume or secret based mechanism for this one.